spec/defines/host_spec.rb

```diff
@@ -29,6 +29,43 @@
           ])
         }
       end
+
+      describe 'comment parameter' do
+        let :params do {
+          :ip      => '10.0.0.100',
+          :mac     => '01:02:03:04:05:06',
+          :comment => 'a useful comment',
+        } end
+
+        let :facts do
+          facts
+        end
```

Comment:
We should pull this to line 6 since it can be shared between both blocks.

Reply:
Great minds...

Reply:
Re-pushed.